cmp/grid/Grid.ts

```diff
@@ -107,14 +107,22 @@ export const [Grid, grid] = hoistCmp.withFactory<GridProps>({
             highlightRowOnClick ? 'xh-grid--highlight-row-on-click' : null
         );
 
-        return fragment(
+        const container = model.experimental['enableFullWidthScroll'] ? vframe : frame;
+
+        return container(
```

Comment:
Can we avoid the extra level of DOM nesting that this appears to introduce?  Previously we were returning a `fragment` at the top level, with a frame and supporting colChooser/filter dialogs inside.  

Could those dialogs be moved to nest within the frame/vframe items (alongside the agGrid) so we still have a single outer (v)frame?

Reply:
Sorry about that- actually didn't mean to change the outer `fragment`- this was in a bad state mid-refactor. Just updated the PR